Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -654,3 +678,135 @@ open class ChatMessageListVC: _ViewController,
         !(touch.view is UIControl)
     }
 }
+
+private extension ChatMessageListVC {
+    // MARK: - Message Updates Helpers
+
+    func handleMessageUpdates(with changes: [ListChange<ChatMessage>], completion: (() -> Void)?) {
```

Comment:
This method is too big and it's really hard to understand what it does. Maybe we should split it in smaller methods, while explaining what they do.

Reply:
They are already broken down into multiple functions. I can try to break it down even more, I'll keep this in mind. But this function is what we had previously in [here](https://github.com/GetStream/stream-chat-swift/blob/develop/Sources/StreamChatUI/ChatMessageList/ChatMessageListView.swift#L183-L247).

I just moved here, and ofc added more logic because of this feature and tried to break it into multiple functions. Each function is explaining what it is doing. But as I said, I'll try to break it down even more 👍

Reply:
I've simplified a lot this method now, especially because most of the logic that was done here, is not needed any more 👍